sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -138,7 +142,11 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
           default: ActiveApplicationsSummary,
           sidebar: InstitutionHomeSideBar,
         },
-        props: true,
+        props: {
+          default: (route: RouteLocationNormalized) => ({
```

Comment:
What is the benefit of having the default here?
Would the below not be enough?
```ts
  props: (route: RouteLocationNormalized) => ({
            locationId: parseInt(route.params.locationId[0])
          }),
        },
```

Reply:
I tried doing that but it didn't accept:

![image](https://user-images.githubusercontent.com/78114138/235990340-046a2053-3b8f-4466-8cea-a7b04fbbb60f.png)

Reply:
I see the point but maybe we are doing something else wrong.
The [function mode](https://router.vuejs.org/guide/essentials/passing-props.html#function-mode) seems to be available and we maybe are missing something about how to have it properly typed.

Either way, I would be ok to proceed as it is right now 😉

Reply:
https://router.vuejs.org/guide/essentials/passing-props.html#named-views

Reply:
> I think the default adds props to the default component, ![image](https://user-images.githubusercontent.com/77353155/235986845-e9e0c599-3f92-40f2-8dcd-605180385fc4.png) and if there are any param for the sidebar component then `props: { sidebar: (route: RouteLocationNormalized) => ({` can also be added. correct me if I am wrong @andrepestana-aot

@ann-aot After our chat I got the point here and I am ok about doing the parse for the individual components. As we talk, for an upcoming PR, where we would be doing the changes globally, I would like to explore some centralized method to avoid repeating code like `route.params.locationId[0]`.
@andrepestana-aot since this was not the main focus of the PR I am ok as it is right now.

Reply:
> > I think the default adds props to the default component, ![image](https://user-images.githubusercontent.com/77353155/235986845-e9e0c599-3f92-40f2-8dcd-605180385fc4.png) and if there are any param for the sidebar component then `props: { sidebar: (route: RouteLocationNormalized) => ({` can also be added. correct me if I am wrong @andrepestana-aot

Yes, that's right.


> @ann-aot After our chat I got the point here and I am ok about doing the parse for the individual components. As we talk, for an upcoming PR, where we would be doing the changes globally, I would like to explore some centralized method to avoid repeating code like `route.params.locationId[0]`. @andrepestana-aot since this was not the main focus of the PR I am ok as it is right now.

Ok. @andrewsignori-aot  I didn't understand well what you mean about a centralized method. Do mean creating a name function to be reused for the named objects?